src/Broadway/Bundle/BroadwayBundle/DependencyInjection/BroadwayExtension.php

```diff
@@ -107,6 +110,31 @@ private function loadEventStore(array $config, ContainerBuilder $container)
             'broadway.event_store.dbal.use_binary',
             $config['dbal']['use_binary']
         );
+
+        $this->dbalEventStoreConnection = $config['dbal']['connection'];
+    }
+
+    /**
+     * This needs to be done in a compiler pass, to access doctrine connections definitions.
+     */
+    public function defineDBALEventStoreConnection(ContainerBuilder $container)
```

Comment:
I would prefer to move this to the actual compiler pass. You can set a parameter for the connection name.

I see you also use `$this->getAlias()`, but we can probably pass it as a constructor argument to the compiler pass or maybe even hardcode it.